src/App.js

```diff
@@ -0,0 +1,26 @@
+import './App.css';
+import { useEffect } from 'react';
+import { useRoutes } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import Sidebar from './Components/SideBar/Sidebar';
+import routes from './config/routes';
+import { checkUser } from './Redux/User/registerSlice';
+
+function App() {
```

Comment:
- [ ] Please, kindly refactor this component declaration with ES6 (arrow) syntax.  

Reason:  Almost the whole software industry has moved away from the `function` keyword.

That way, your code will comply with latest standards 💪

Reply:
- [x] Done, thanks for pointing this out 👍